WeatherApp/app/src/main/java/com/example/weatherapp/MainActivity.kt

```diff
@@ -37,6 +36,7 @@ class MainActivity : AppCompatActivity() {
                                     Manifest.permission.ACCESS_FINE_LOCATION) ==
                                     PackageManager.PERMISSION_GRANTED)) {
                         Toast.makeText(this, resources.getString(R.string.permission_granted), Toast.LENGTH_SHORT).show()
+                       // getLocation()
```

Comment:
@corinaorasan171015 Commented code on the main branch can remain there forever. Let's remove this line if it's needed only for local debug.